src/common/models/Projection.ts

```diff
@@ -394,88 +385,44 @@ export class Projection {
     return s.countyName ? `${s.countyName}, ${s.stateName}` : s.stateName;
   }
 
-  /**
-   * Gets the cases/day on day i, by averaging the trailing week of cases/day
-   * data.
-   *
-   * Note: In the case of a data point with negative cases/day (can happen due
-   * to reporting weirdness), we skip that data point and may average fewer
-   * than 7 days worth of data.
-   */
-
-  getWeeklyAverageForDay(data: Array<number | null>, i?: number) {
-    const lastIndex = indexOfLastValue(data);
-    if (i === undefined) {
-      if (!lastIndex) return null;
-      i = lastIndex;
-    } else if (lastIndex === null || i > lastIndex) {
-      return null;
-    }
-    // Get last week of sane data (ignore negative values)
-    const lastWeekOfPositives = _.filter(
-      data.slice(Math.max(0, i - 6), i + 1),
-      p => p !== null && p >= 0,
-    );
-    return _.mean(lastWeekOfPositives);
-  }
-
   private calcContactTracers(
     actualTimeseries: Array<CANActualsTimeseriesRow | null>,
   ): Array<number | null> {
-    // use date.getWeek() to get the week number
-    return actualTimeseries.map(
-      (row: CANActualsTimeseriesRow | null, i: number) => {
-        if (row && row.contactTracers) {
-          const contactTracers =
-            CONTACT_TRACER_STATE_OVERRIDES[this.stateName] ||
-            row.contactTracers;
-          const weeklyAverage = this.getWeeklyAverageForDay(
-            this.dailyPositiveTests,
-            i,
-          );
-
-          if (weeklyAverage) {
-            return contactTracers / (weeklyAverage * TRACERS_NEEDED_PER_CASE);
-          }
+    return actualTimeseries.map((row, i) => {
+      if (row && row.contactTracers) {
+        const contactTracers =
+          CONTACT_TRACER_STATE_OVERRIDES[this.stateName] || row.contactTracers;
+        const cases = this.smoothedDailyPositiveTests[i];
+        if (cases) {
+          return contactTracers / (cases * TRACERS_NEEDED_PER_CASE);
         }
+      }
```

Comment:
Should we return 100% if `cases = 0`? It's a bit of an edge case. We probably don't need to worry about it since we are using the smooth series now.

Reply:
Oh, that's a good question.  Right now we only show contact tracing at the state-level, so this hasn't come up for current data (I don't think any state has reached 0 cases)...  

I'm going to leave it as-is for now...  In the case that we start showing contact tracing for counties (which may be more expected to have zero case counts), I'm not sure if it makes sense to omit the metric or show 100% when they haven't had any cases (yet).